File: Excel_Programs/C++/question1.cpp
//Reverse of array
#include<iostream>
using namespace std ;

int main()
{
    int n;
    cout<<"Enter the number of elements to be entered : ";
    cin>>n;
    int arr[n];
    int arr2[n];
    for(int i =0; i<n;i++)
    {
        cout<<"Enter a number : ";
        cin>>arr[i];
    }
    cout<<"The elements entered in the array \n";
    for(int i=0;i<n;i++)
    {
        cout<<arr[i];
        cout<<" ";
    }
    cout<<"\n";
    for(int i=0;i<n;i++)
    {
        arr2[i] = arr[n-i-1];
    }
    cout<<"The reverse of the array : ";
    for (int i =0;i<n;i++)
    {
        cout<<arr2[i];
        cout<<" ";
    }
    cout<<"\n";

    return 0;

}

File: Excel_Programs/Python/question1.py
#Reverse of array
n = int(input("Enter the number of numbers to be entered : "))
arr=[]
arr2=[]
for i in range(n):
    arr.append(int(input("Enter a number : ")))
print()
print("The numbers entered in the array : ")
for i in range(n):
    print(arr[i],end=" ")
print()

for i in range(n):
    arr2.append(arr[n-i-1])

print("The reverse of the array : ")
for i in range(n):
    print(arr2[i],end=" ")
print()